en/uavcan/ark_rtk_gps.md

```diff
@@ -0,0 +1,135 @@
+# ARK RTK GPS
+
+ARK RTK GPS is an open source [UAVCAN](README.md) [RTK GPS](../gps_compass/rtk_gps.md), [u-blox F9P](https://www.u-blox.com/en/product/zed-f9p-module), magnetometer, barometer, IMU, buzzer, and safety switch module.
+
+![ARK RTK GPS](../../assets/hardware/gps/ark_rtk_gps.jpg)
```

Comment:
This image is 1.35Mb, which is pretty large for the content. Can you shrink it a bit?
For example, this cuts off the dead space and reduces width to 1000px - only 140Kb ish.
![ark_rtk_gps](https://user-images.githubusercontent.com/5368500/145141526-9f25291f-d8cd-4ee7-8aeb-9391b52b732d.jpg)